src/commands/kv/namespace/site.rs

```diff
@@ -63,6 +62,15 @@ fn get_id_from_namespace_list(
             .find(|ns| ns.title == title)
             .unwrap()
             .to_owned()),
-        Err(e) => failure::bail!(kv::format_error(e)),
+        Err(e) => failure::bail!("{}", http::format_error(e, Some(&error_suggestions))),
```

Comment:
kv  actually has a format  error fn:  https://github.com/cloudflare/wrangler/blob/josh/fix-broken-publish/src/commands/kv/mod.rs#L35 try  using/adding to t hat.